README.md

```diff
@@ -8,13 +8,13 @@ This is an API that [creates AWS (sub)accounts in an Organization](https://docs.
 
 1. Install system dependencies.
     1. [Go](https://golang.org/)
-    1. [Dep](https://golang.github.io/dep/docs/installation.html)
+    1. [Dep](https://golang.github.io/dep/docs/installation.html) (`go get -u github.com/golang/dep/cmd/dep`)
 1. Clone the repository.
 
     ```sh
-    cd $(go env GOPATH)/src
-    mkdir -p GSA
-    cd GSA
+    export GOPATH=~/go # or whatever go workspace you prefer
```

Comment:
As of Go 1.8, the default GOPATH is `~/go` unless otherwise set (which is what `go env GOPATH`), so was leveraging that. Is it still needed, you think?

https://golang.org/doc/code.html#GOPATH

Reply:
Yes, but without that, you can't just cut-and-paste.  In my experience, I get fewer people telling me the instructions didn't work when I make it easy for them to cut and paste.  Is the `github.com` part of the path necessary or not?

Reply:
> without that, you can't just cut-and-paste

Without Go 1.8+, you mean?

> Is the `github.com` part of the path necessary or not?

Ooh, shoot, you're right - that was a typo on my part.